be/src/common/config.h

```diff
@@ -1215,4 +1215,7 @@ CONF_mBool(enable_profile_for_external_plan, "false");
 
 // the max length supported for varchar type
 CONF_mInt32(olap_string_max_length, "1048576");
+
+// jit LRU cache size for total 32 shards.
+CONF_mInt64(jit_lru_cache_size, "1073741824");
```

Comment:
1G may be too large if BE deployed on a small instance, should we support set a ratio for it? just like storage_page_cache_limit

Reply:
if the cache size is too smaller, the cache becomes invalid, possiblely introducing some bad cases. We can disable jit if the system memory or process  mem limit is less than X bytes.